docs/openapi.yaml

```diff
@@ -1672,23 +1674,41 @@ components:
           type: string
           description: >
             The password to use when logging into the fish node.
+        key:
+          type: string
+          description: >
+            SSH key could be used instead of password to access the system.
+        port:
+          type: integer
+          description: >
+            TCP port to connect with SSH client.
```

Comment:
Are we sure `integer` is correct here?  From what I could see, all of the tests are using `:0` for their port, which won't cause any issues.  This is something about networking I'm not very knowledgeable about; usually, there's up to four digits allowed.  Specifically for anything less than 1000, should a user be able to supply e.g., `0999` and have it work?  Or is requiring `999` (or `99` rather than `0099`) ok?

If the leading zeros are supposed to be allowed, I think we have to make this `type: string`.  If leading zeros aren't a concern, then we can disregard!

Reply:
Tests are using :0 because this will tell the OS to assign a random not-busy port for binding. And this port in the yaml is to specify where to connect on the target remote system.

And sorry, but I did not get the question at all... IP Port range is uint16 from 1 to 65535 (as per IP specs), but for simplicity I use here regular integer, because openapi don't care about how much bytes is there encoding the number.

As per wikipedia:
> 0 - In programming APIs (not in communication between hosts), requests a system-allocated (dynamic) port

Reply:
But this is made me to think - why ResourceAccess has port in it... And for some reason I added it here and there just for fun) So removed from ResourceAccess at all since not used.